server/auth/login.py

```diff
@@ -38,7 +40,16 @@ def login_request(self, packet, address):
             auth_status = 'bad_credentials'
             uid = None
         else:
-            uid = self.server.handle_until_value('auth:get_user_id', True, address)
+            user = authenticate(username=packet.username, password=packet.password)
```

Comment:
Leave it as it was. This directly couples this code to the DB, as opposed to delegating to the event bus. The issue is that it was using handle_until_value instead of handle_until_return